bee-message/src/message.rs

```diff
@@ -220,3 +220,70 @@ fn verify_payload(payload: Option<&Payload>) -> Result<(), Error> {
         Ok(())
     }
 }
+
+#[cfg(feature = "dto")]
+#[allow(missing_docs)]
+pub mod dto {
+    use serde::{Deserialize, Serialize};
+
+    use super::*;
+    use crate::{error::dto::DtoError, payload::dto::PayloadDto};
+
+    /// The message object that nodes gossip around in the network.
+    #[derive(Clone, Debug, Serialize, Deserialize)]
+    pub struct MessageDto {
+        ///
+        #[serde(rename = "protocolVersion")]
+        pub protocol_version: u8,
+        #[serde(rename = "parentMessageIds")]
+        ///
+        pub parents: Vec<String>,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        ///
+        pub payload: Option<PayloadDto>,
+        ///
+        pub nonce: String,
```

Comment:
missing docs :trollface:

Reply:
Yeah not doing it in this PR, simply moving stuff around.